### PR TITLE
style(GPC logs): Style guide edits

### PR DESCRIPTION
**Pull request by barbnewrelic**
WIP: Update google-cloud-platform-log-forwarding, move headerless section into a collapser

Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

File: src/content/docs/logs/forward-logs/google-cloud-platform-log-forwarding.mdx [hunk 1]
```diff
@@ -7,373 +7,450 @@ tags:
...
-      </td>
-
       <td>
-        N/A
-      </td>
-
-      <td>
-        The Cloud Pub/Sub subscription to consume from. The name should be in the format of `projects/<project-id>/subscriptions/<subscription-name>`
+        * Best suited for larger log volumes, since it groups log records in batches before sending them to New Relic.
+        * Reduces the number of API calls and allows to reduce your quota usage.
```
Comment (mmfred) [inline src/content/docs/logs/forward-logs/google-cloud-platform-log-forwarding.mdx:48]:
```suggestion
        * Reduces the number of API calls and allows you to reduce your quota usage.
```

File: src/content/docs/logs/forward-logs/google-cloud-platform-log-forwarding.mdx [hunk 2]
```diff
@@ -7,373 +7,450 @@ tags:
...
+  <Collapser
+    id="create-gcp-topic"
+    title="2. Create a GCP Pub/Sub Topic."
+  >
 
-    <tr>
-      <td>
-        flushDelay
-      </td>
+  Next you will create the GCP Pub/Sub topic that your ingest URL will use.
```
Comment (mmfred) [inline src/content/docs/logs/forward-logs/google-cloud-platform-log-forwarding.mdx:84]:
```suggestion
  Next you'll create the GCP Pub/Sub topic that your ingest URL will use.
```

File: src/content/docs/logs/forward-logs/google-cloud-platform-log-forwarding.mdx [hunk 3]
```diff
@@ -7,373 +7,450 @@ tags:
...
+To send your GCP logs to New Relic using a Dataflow job, you will use our Dataflow template. Before you begin, make sure you have the following tools on your local computer:
 
-      <td>
-        Compress (in GZIP) the payloads sent to the New Relic Logs API.
-      </td>
-    </tr>
+* A Unix terminal for Linux or MacOS
+* [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+* [Java JDK 8 or higher](https://adoptopenjdk.net/index.html)
+* [Apache Maven 3.2 or higher](https://maven.apache.org/). Earlier versions have been observed to fail during the compilation process.
```
Comment (mmfred) [inline src/content/docs/logs/forward-logs/google-cloud-platform-log-forwarding.mdx:142]:
```suggestion
* [Apache Maven 3.2 or higher](https://maven.apache.org/). We've seen earlier versions fail during the compilation process.
```

File: src/content/docs/logs/forward-logs/google-cloud-platform-log-forwarding.mdx [hunk 4]
```diff
@@ -7,373 +7,450 @@ tags:
...
+    title="2. Clone the DataflowTemplates GitHub repository."
+  >
+
+  1. Clone the DataflowTemplates GitHub repository using the following command:
+
+  ```
+  git clone https://github.com/newrelic-forks/DataflowTemplates.git
+  ```
+
+  2. Enter the created directory to execute the following commands:
```
Comment (mmfred) [inline src/content/docs/logs/forward-logs/google-cloud-platform-log-forwarding.mdx:172]: commands or command? Looks like just one.

File: src/content/docs/logs/forward-logs/google-cloud-platform-log-forwarding.mdx [hunk 5]
```diff
@@ -7,373 +7,450 @@ tags:
...
+        </td>
+      </tr>
+
+      <tr>
+        <td>
+          `logsApiUrl`
+        </td>
+
+        <td>
+          New Relic's Log API url. This should be routable from the VPC in which the Dataflow pipeline runs.
```
Comment (mmfred) [inline src/content/docs/logs/forward-logs/google-cloud-platform-log-forwarding.mdx:363]:
```suggestion
          New Relic's Log API URL. This should be routable from the VPC in which the Dataflow pipeline runs.
```

File: src/content/docs/logs/forward-logs/google-cloud-platform-log-forwarding.mdx [hunk 6]
```diff
@@ -7,373 +7,450 @@ tags:
...
+        </td>
+      </tr>
+
+      <tr>
+        <td>
+          `batchCount`
+        </td>
+
+        <td>
+          Maximum number of log records to aggregate into a batch before sending them to NewRelic in a single HTTP POST request.
```
Comment (mmfred) [inline src/content/docs/logs/forward-logs/google-cloud-platform-log-forwarding.mdx:379]:
```suggestion
          Maximum number of log records to aggregate into a batch before sending them to New Relic in a single HTTP POST request.
```

File: src/content/docs/logs/forward-logs/google-cloud-platform-log-forwarding.mdx [hunk 7]
```diff
@@ -7,373 +7,450 @@ tags:
...
+  </Collapser>
+
+  <Collapser
+    id="dataflow-config"
+    title="8. Optional: Tune up your Dataflow log forwarder job."
+  >
+
+  Here is a reference of available options you can use to further tune up the execution of your Dataflow log forwarder job.
+
+  <table>
```
Comment (mmfred) [inline src/content/docs/logs/forward-logs/google-cloud-platform-log-forwarding.mdx:317]: Consider expanding the column width for that first column in the following table - some of the parameter names are pretty smushed, and the descriptions will more easily wrap.

**Review by mmfred — CHANGES_REQUESTED** [review]
@barbnewrelic a few small things.

**Comment by barbnewrelic** [comment]
PR review suggestions incorporated. Merging this PR

**Pull request merged**